packages/teams-js/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["node_modules", "./webpack.config.ts"]
```

Comment:
Eslint shares the main tsconfig except that it does NOT exclude the test directory, since we want eslint to run on the test directory as well.